ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/PutJobManagement_Test.java

```diff
@@ -0,0 +1,556 @@
+
+            final Ds3ClientHelpers.Job jobOne =
+                    HELPERS.startWriteJob(BUCKET_NAME, Lists.newArrayList(new Ds3Object("test", 2)), writeJobOptions);
+            final UUID jobOneId = jobOne.getJobId();
+
+            final Ds3ClientHelpers.Job jobTwo =
+                    HELPERS.startWriteJob(BUCKET_NAME, Lists.newArrayList(new Ds3Object("test2", 2)), writeJobOptions);
+            final UUID jobTwoId = jobTwo.getJobId();
+
+            assertThat(jobOneId, is(jobTwoId));
+
+        } finally {
+            deleteAllContents(client, BUCKET_NAME);
+        }
+    }
+
+    @Test
+    public void allocateJobChunk() throws IOException, SignatureException, XmlProcessingException {
+        
+        try {
+            final PutBulkJobSpectraS3Response putBulkResponse = client.
+                    putBulkJobSpectraS3(new PutBulkJobSpectraS3Request(BUCKET_NAME, Lists.newArrayList(new Ds3Object("test", 2))));
+            final UUID chunkUUID = putBulkResponse.getResult().getObjects().get(0).getChunkId();
+            final AllocateJobChunkSpectraS3Response allocateResponse = client
+                    .allocateJobChunkSpectraS3(new AllocateJobChunkSpectraS3Request(chunkUUID));
+
+            assertThat(allocateResponse.getStatusCode(), is(200));
+
+        } finally {
+            deleteAllContents(client, BUCKET_NAME);
+        }
+    }
+
+    @Test
+    public void putObjectCachedNotification() throws IOException, SignatureException, XmlProcessingException {
+        try {
+            final PutObjectCachedNotificationRegistrationSpectraS3Response putNotificationResponse = client
+                    .putObjectCachedNotificationRegistrationSpectraS3
+                            (new PutObjectCachedNotificationRegistrationSpectraS3Request("test@test.test"));
+
+            assertThat(putNotificationResponse.getStatusCode(), is(201));
+
+        } finally {
+            deleteAllContents(client, BUCKET_NAME);
+        }
+    }
+
+    @Test
+    public void getObjectCachedNotification() throws IOException, SignatureException, XmlProcessingException {
+
+        try {
+            final PutObjectCachedNotificationRegistrationSpectraS3Response putNotificationResponse = client
+                    .putObjectCachedNotificationRegistrationSpectraS3
+                            (new PutObjectCachedNotificationRegistrationSpectraS3Request("test@test.test"));
+            final GetObjectCachedNotificationRegistrationSpectraS3Response getNotificationResponse = client
+                    .getObjectCachedNotificationRegistrationSpectraS3(
+                            (new GetObjectCachedNotificationRegistrationSpectraS3Request
+                                    (putNotificationResponse.getS3ObjectCachedNotificationRegistrationResult().getId())));
+            assertThat(getNotificationResponse.getStatusCode(), is(200));
+
+        } finally {
+            deleteAllContents(client, BUCKET_NAME);
```

Comment:
It doesn't look like you are doing anything with the bucket here.  This can be removed.